digiroad2-oracle/sql/export_road_addresses.sql

```diff
@@ -154,10 +154,10 @@ BEGIN
 				INSERT INTO EXPORT_TABLE VALUES (COUNTER,
```

Comment:
Is this table modified also?

Reply:
Tää on Andren skripti jolla tuodaan teitä omaan kantaan. Se luo insert lauseita tuonne export tauluun.